crates/common/src/traits.rs

```diff
@@ -4,6 +4,35 @@ use alloy_json_abi::Function;
 use alloy_primitives::Bytes;
 use alloy_sol_types::SolError;
 use std::{fmt, path::Path};
+use tracing::warn;
+
+/// List of functions included with Foundry that are prefixed with 'assert' but are not intended to be used as assertions.
+static FOUNDRY_ASSERTION_FN_NAMES: &[&str] = &[
```

Comment:
Does the test runner pick up on these? I wouldn't think so since they are internal functions.